src/main/java/com/minecolonies/coremod/entity/ai/citizen/guard/GuardArrow.java

```diff
@@ -61,7 +67,7 @@ protected void arrowHit(final EntityLivingBase targetEntity)
     {
         super.arrowHit(targetEntity);
         Log.getLogger().info("Arrow hit " + targetEntity + " with " + targetEntity.getHealth());
-        if (targetEntity.getHealth() <= 0.0F)
+        if (targetEntity.getHealth() <= 0.0F && !(shootingEntity instanceof EntityArcherBarbarian))
```

Comment:
Could you explain shootingEntity instanceof EntityArcherBarbarian to me ? why that ?